### ISSUE TITLE
[Idea] Host a panel conversation about POCs and disabilities

### ISSUE THREAD

**Issue by nickttng** [A11YChi/meetups-and-events#21]
**Why:** A panel about intersectionality between races and disabilities, including the topic - up to 50% of people with disabilities, and majority of them are black, are killed by police. Accessibility professionals and advocates cannot turn away from this fact, and need to be aware of it.  

**How:** Seek out panelists and/or orgs that focus on this issue, and identify steps from there.
